### PR TITLE
first draft of citation file

### PR DESCRIPTION
**Pull request by shimwell**
## Description
Following on from discussion with @kkiesling in the Slack channel here is a draft citation file.

I have been through the contributors section and added details for each github username that has contributed. This is difficult for a few user accounts that don't have first or last names in their profile. Perhaps these users can be tagged using @ if this PR goes ahead.
ORCID are also difficult to find for everyone so perhaps the contributors with missing ORCID can also be tagged and asked for their ORCID.

Currently the list is sorted by commits as that was easy to get from the contributors page, but could be changed to any order that is preferred?
Also a paper could be added if that is preferred?

Further details 

## Motivation and Context
Helps users easily cite the software using the standard GitHub CITATION file which produces a bibtex and other formats citation link on the right hand side of the repository. See here for more details https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files

## Changes
documentation update

## Behavior
Citation of the software is perhaps less frequent and harder than it needs to be

### CODE REVIEW

**Comment by kkiesling** [comment]
This has prompted me to find my ORCID that I made a long time ago and haven't ever used.. https://orcid.org/0000-0002-1215-2541

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+- family-names: "Wilson"
+  given-names: "Paul"
+  orcid: "https://orcid.org/0000-0002-8555-4410"
+- family-names: "Harb"
+  given-names: "Moataz"
+  orcid: "https://orcid.org/0000-0002-0037-9087"
+- family-names: "Dunn"  
+  given-names: "Kerry"
+- family-names: "Kiesling" 
+  given-names: "Kalin"
```
Comment (shimwell) [inline CITATION.cff:23]:
```suggestion
- family-names: "Kiesling" 
  given-names: "Kalin"
  orcid: "https://orcid.org/0000-0002-1215-2541"
```

**Comment by pshriwise** [comment]
@iulian787 and @vijaysm  are almost certainly Iulian Grindeanu (https://orcid.org/0000-0002-0264-8236) and Vijay S. Mahadevan (https://orcid.org/0000-0002-3337-2607) from ANL.

It looks like @SudoVim is from pullrequest.com?

**Comment by SudoVim** [comment]
> It looks like @SudoVim is from pullrequest.com?

Yeah, I'm from pullrequest.com. Do you still need my last name? At any rate, I updated my profile with my company and name.

**Comment by shimwell** [comment]
> > It looks like @SudoVim is from pullrequest.com?
> 
> Yeah, I'm from pullrequest.com. Do you still need my last name? At any rate, I updated my profile with my company and name.

Super, thanks. I shall add your names as it is on the profile

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  given-names: "Elliott"
+  orcid: "https://orcid.org/0000-0002-9088-1360"
+- family-names: "D'Angelo"
+  given-names: "Chelsea"
+  orcid: "https://orcid.org/0000-0001-8852-0366"
+- family-names: "Schlömer"
+  given-names: "Nico"
+# # @SudoVim unkown familiy/given name
+# - family-names: ""
+#   given-names: ""
```
Comment (shimwell) [inline CITATION.cff]:
```suggestion
 Avatar
 - family-names: "Renken"
   given-names: "Michael"
```

File: CITATION.cff [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  orcid: "https://orcid.org/0000-0002-1147-045X"
+- family-names: "Biondo" 
+  given-names: "Elliott"
+  orcid: "https://orcid.org/0000-0002-9088-1360"
+- family-names: "D'Angelo"
+  given-names: "Chelsea"
+  orcid: "https://orcid.org/0000-0001-8852-0366"
+- family-names: "Schlömer"
+  given-names: "Nico"
+ Avatar
```
Comment (shimwell) [inline CITATION.cff]:
```suggestion
```

**Comment by shimwell** [comment]
I think we have nearly everyone who has contributed apart

Just missing the name for @dqwu

**Review by pshriwise — APPROVED** [review]
This has been sitting here for a while. It looks like a good addition to me. Thanks @shimwell!

**Pull request merged**